src/main/java/org/sonatype/central/publisher/plugin/AbstractPublisherMojo.java

```diff
@@ -27,6 +27,15 @@ public abstract class AbstractPublisherMojo
   @Parameter(property = "skipPublishing", defaultValue = "false", required = true)
   private boolean skipPublishing;
 
+  /**
+   * For skipping artifacts bundling.
+   +
+   + This is particularly useful for multi-module build, where some sub-modules should
+   + not be included in the final bundle that will be uploaded to <code>central.sonatype.com</code>.
+   */
```

Comment:
I did not add the `@since` version, since I don't know what will be the next version (1.2.0?).